src/main/java/org/jitsi/videobridge/xmpp/MediaStreamTrackFactory.java

```diff
@@ -86,31 +96,51 @@
      * experimental and is left disabled by default.
      */
     private static final boolean ENABLE_VP9_SVC
-        = cfg.getBoolean(ENABLE_VP9_SVC_PNAME, false);;
+        = cfg.getBoolean(ENABLE_VP9_SVC_PNAME, false);
 
     /**
      * A boolean that's controlling whether or not to enable SVC filtering for
      * scalable video codecs.
      */
-    private static final boolean ENABLE_SVC
+    private static final Boolean ENABLE_SVC
         = cfg.getBoolean(ENABLE_SVC_PNAME, false);
 
+    /**
+     * libjitsi isn't aware of the group semantics names defined in
+     * {@link SourceGroupPacketExtension}, which is how we distinguish secondary
+     * ssrcs, so we'll translate them into constants defined in libjitsi
+     */
+    private static Map<String, String> secondarySsrcTypeMap = null;
+
+    private static synchronized Map<String, String> getSecondarySsrcTypeMap()
+    {
+        if (secondarySsrcTypeMap == null)
+        {
+            secondarySsrcTypeMap = new HashMap<>();
+            secondarySsrcTypeMap.put(SourceGroupPacketExtension.SEMANTICS_FID, Constants.RTX);
```

Comment:
Do we have support for flex-fec SSRCs?

Reply:
not yet, this PR doesn't have the flexfec work (just one of the base constants) that's coming in later PR